pkg/kapp/cmd/app/apply_flags.go

```diff
@@ -45,6 +45,8 @@ func (s *ApplyFlags) SetWithDefaults(prefix string, defaults ApplyFlags, cmd *co
 		mustParseDuration("15m"), "Maximum amount of time to wait in apply phase")
 	cmd.Flags().DurationVar(&s.ApplyingChangesOpts.CheckInterval, prefix+"apply-check-interval",
 		mustParseDuration("1s"), "Amount of time to sleep between applies")
+	cmd.Flags().BoolVar(&s.ApplyingChangesOpts.Run, prefix+"apply-run", false,
+		"Print new generated yaml to be applied and exit. This flag can not be used with others.")
```

Comment:
"yaml" should be uppercased since it's an acronym.

Reply:
> This flag can not be used with others.

well, that's not quire right that it cannot be used (plenty of flags do their thing with it). i would recommend dropping this.

Reply:
updated